src/pasta/fields/fq.rs

```diff
@@ -164,69 +164,61 @@ impl_binops_additive!(Fq, Fq);
 impl_binops_multiplicative!(Fq, Fq);
 
 /// INV = -(q^{-1} mod 2^64) mod 2^64
-const INV: u64 = 0x842cafd3ffffffff;
+const INV: u64 = 0x8c46eb20ffffffff;
```

Comment:
I did not check this constant, but I am satisfied by its unit test.

Reply:
Checked up to here.